### PR TITLE
Verbs cleanup

### PR DESCRIPTION
**Pull request by emmetthitz**
Dependent on the work in #1179. Once that gets merged in, this should be much smaller.

### CODE REVIEW

**Comment by dkoshkin** [comment]
The docs and integration-test specs need to be modified, find references of `kismatic install`

**Comment by emmetthitz** [comment]
Oh wait, I see what you mean. Working on it.

**Comment by emmetthitz** [comment]
Should be everything.

File: docs/install.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 * Provision infrastructure (make machines or alter your network)
 * Install load balancers
 
-# Kismatic Install Workflow
+# kismatic Workflow
```
Comment (dkoshkin) [inline docs/install.md]: Capitalize

File: docs/intent.md [hunk 2]
```diff
@@ -96,7 +96,7 @@
      * sudo access to root
      * Public Key auth for sudo & sshd
      * The public key generated above in `~/.ssh/id_rsa.pub`
-   * You may choose to [pre-install](provision.md#pre-install-configuration) packages, or let Kismatic install them for you.
```
Comment (dkoshkin) [inline docs/intent.md]: This should actually stay the same

File: docs/intent.md [hunk 3]
```diff
@@ -108,7 +108,7 @@
...
```
Comment (dkoshkin) [inline docs/intent.md]: Don't change

File: docs/intent.md [hunk 4]
```diff
@@ -171,7 +171,7 @@
...
```
Comment (dkoshkin) [inline docs/intent.md]: Don't change

File: docs/intent.md [hunk 5]
```diff
@@ -181,7 +181,7 @@
...
```
Comment (dkoshkin) [inline docs/intent.md]: Don't change

File: docs/intent.md (hunk 2 above)
Comment (emmetthitz) [reply docs/intent.md]: Whoops 😅

**Comment by emmetthitz** [comment]
Fixed!

**Comment by dkoshkin** [comment]
LGTM!

**Pull request merged**